frontend/cypress/e2e/talentsearch/support-page.cy.js

```diff
@@ -0,0 +1,33 @@
+describe("Support page", () => {
+
+  const name = "Test Person"
+  const email = "test@test.tld"
+  const details = "Test comments to send."
+  const subject = "question"
+
+  context('Support page', () => {
+    it('should exist', () => {
+      cy.visit('/en/support')
+      cy.findByRole('heading', { name: 'Contact and support', level: 1 }).should('exist')
+    })
+  })
+
+  context('Support form', () => {
+    it("should send POST request to existing api endpoint", () => {
+      cy.request({
+        method: "POST",
+        url: "/api/support/tickets",
+        failOnStatusCode: false,
+        body: {
+          name,
+          email,
+          details,
+          subject
+        },
+      }).then((response) => {
+        if(response.status === 404) // verify if route exists.
```

Comment:
Ya, not ideal, was racking by brain on what the best assertion logic would be but settled on throwing an error for if the route did not exist (i.e. 404). What comment would you suggest adding?

I also ran into problems trying to fill out the form and intentionally abandoned that part of the test because **react-select** is super hard to test reliably.

Reply:
Ideally we'd have a mock for the service locally that responds in a realistic way.  Someday..

Reply:
This looks great! Using react-select is best done via keyboard nav in my experience. Might be worth create helpers down the road, to select based on keyboard input, like so:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/2f2c33346b1f399bd71d4a5a3372c6f8167ec998/frontend/common/src/components/form/MultiSelect/MultiSelectFieldV2.test.tsx#L53-L56